src/main/java/com/microsoftopentechnologies/windowsazurestorage/AzureStorageBuilder.java

```diff
@@ -273,12 +274,16 @@ public synchronized void perform(
                 }
             }
 
+            // Resolve container name or share name
+            final String expContainerName = Utils.replaceMacro(Util.fixNull(containerName), envVars, Locale.ENGLISH);
```

Comment:
why hardcode engglish locale? whta's container name and share name's naming covention?